portafly/README.md

```diff
@@ -1,6 +1,31 @@
 # PortaFly
 A React SPA crafted with the ambitious purpose of replacing Porta UI, currently residing inside the Rails App.
 
+## Getting up and running
+1. Clone the `porta` repository to your local machine
+2. Create a `.env` file:
+  * Create a new file in the `/portafly` root folder, and name it `.env`
+  * Paste `REACT_APP_API_HOST="https://multitenant-admin.preview01.3scale.net"` into it, and Save
+3. Create an Access token:
+  * Access your local Porta build (or access [Preview](https://multitenant-admin.preview01.3scale.net/))
+  * Navigate to Settings > Personal > Tokens
+  * Click the "Add Access Token" button
+  * Give your access token a Name
+  * Select all scopes
+  * Set permissions to Read & Write
+  * Create the access token
+  * Copy the access token
+4. Allow CORS requests in your browser:
+  * Install the CORS browser extension ([Google Chrome extension](https://chrome.google.com/webstore/detail/allow-cors-access-control/lhobafahddgcelffkeicbaginigeejlf))
```

Comment:
It is ok to reference to the chrome extension, but imo, we should also reference to [the firefox one](https://addons.mozilla.org/en-US/firefox/addon/cors-everywhere/).
specially now that we are considering running our cukes with firefox: https://github.com/3scale/porta/pull/2039

Reply:
agreed, thank you for linking that!